pkg/image/validation/version.go

```diff
@@ -0,0 +1,29 @@
+package validation
+
+import (
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+)
+
+const (
+	versionComponent = "Version"
+)
+
+func validateVersion(ctx *image.Context) []FailedValidation {
+	var failures []FailedValidation
```

Comment:
Might not be a bad idea to drop a comment in here indicating this is a temporary implementation until we find something cleaner. That way someone new to the codebase won't be as confused.